src/lib/components/SettingsModal.svelte

```diff
@@ -75,6 +75,14 @@
 					</ul>
 				</div>
 			{/if}
+			<label class="flex cursor-pointer select-none items-center gap-2 text-sm text-gray-500">
+				<input
+					type="checkbox"
+					name="hideEmojiOnSidebar"
+					bind:checked={settings.hideEmojiOnSidebar}
+				/>
+				Hide emoji on sidebar
```

Comment:
nit but i's not only is it in the sidebar, it's also in the top-level mobile navigation.

Reply:
I think my proposed wording was fine: "Show emoticons in conversation topics"

Reply:
Updated the wording!